core/route/allrouting.js

```diff
@@ -87,6 +87,18 @@ class Routing {
     }
 
     /**
+     * Match Method
+     * @param {String[]} methods - Array of methods to use for a route
+     * @param {String} path - Path for routing
+     * @param {String|Function} controllerAction
+     */
+    match(methods, path, controllerAction) {
+        methods.forEach((method) => {
+            this[method.toLowerCase()](path, controllerAction);
```

Comment:
Yeah, true. For now I'd just say make a class that extends `Error` like `IllegalMethodError` and just throw that. We can do more with catching that later.